package.json

```diff
@@ -107,7 +107,7 @@
     "workbox-webpack-plugin": "^6.4.1"
   },
   "scripts": {
-    "start": "HTTPS=true node scripts/start.js",
+    "start": "HTTPS=true REACT_APP_ENV=ira-quality-install-tool node scripts/start.js",
```

Comment:
I suggest having this start script use quality-install-tool and then creating a second, ira-start that uses ira-quality-install-tool.

Reply:
Done